client/containers/Pub/pub.scss

```diff
@@ -44,14 +44,18 @@ $breakout-simplify-cutoff: $compact-cutoff-width / (1 - 2 * $pub-margin-fraction
         $comments-full-width
         minmax($pub-margin, auto);
     grid-template-areas: $areas;
-    .breakout {
-        width: 100vw !important;
-        margin-left: 0;
-        position: relative;
-        left: calc(-1 * (50vw - (100% + #{$gutter-full-width} + #{$comments-full-width}) / 2));
+    [data-align='breakout'] {
+        @for $i from 1 through 100 {
+            &[data-size='#{$i}'] {
+                width: $i * 1vw;
+                margin-left: 0;
```

Comment:
I am worried about statically generating this much CSS in a `@for` loop for reasons we're discussing offline, but one small improvement we could make here is just moving the constant declarations, i.e.
```
margin-left: 0;
position: relative;
```
unless Sass has loop optimizations I'm not aware of, that should reduce the impact on the bundle size.

Reply:
Handled in d6864b3.